packages/shared/src/utils/hooks/mounted.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const useIsMounted = () => {
+    const is_mounted = React.useRef(false);
+
+    React.useEffect(() => {
+        is_mounted.current = true;
+
+        return () => (is_mounted.current = false);
+    }, []);
+    return is_mounted;
```

Comment:
You're still returning the ref here rather than the value of the `current` property. You should:

```suggestion
return is_mounted?.current;
```